### ISSUE TITLE
Peer Review for Pseudocode

### ISSUE THREAD

**Issue by skupp32** [Tripfantasy/Demultiplex#5]
if qual average for the score is below cutoff- send to mismatch?
        convert(phred) used:
        exp output: qual score per char in qual score line. Sum and average, then pass to conditional. 

Just one quick thing that Leslie told me.  It's probably better to look specifically at each score and check for a low quality on a base by base basis.  It could be slightly insensitive to too low quality scores unless you set the cutoff so high that you remove a lot of your data.  On so many sequence lines, it might be slightly more efficient if you don't have to check every base in the barcode too- you could potentially decide .  Other than that,  I think you've a good start once you get all the file logic straightened out.

**Comment by Tripfantasy** [comment]
Good point! I'll definitely implement that. Can totally see how using the average can mask low quality scores. Thanks!
